erpnext/accounts/report/financial_statements.py

```diff
@@ -324,12 +324,17 @@ def get_additional_conditions(from_date, ignore_closing_entries, filters):
 		additional_conditions.append("posting_date >= %(from_date)s")
 
 	if filters:
-		for key in ['cost_center', 'project']:
-			if filters.get(key):
-				additional_conditions.append("%s = '%s'"%(key, filters.get(key)))
+		if filters.get("project"):
+			additional_conditions.append("project = '%s'"%(filters.get("project")))
+		if filters.get("cost_center"):
+			additional_conditions.append("%s"%(get_cost_center_cond(filters.get("cost_center"))))
```

Comment:
You can simply write `additional_conditions.append(get_cost_center_cond(filters.get("cost_center")))`